rust/hook-janitor/src/webhooks.rs

```diff
@@ -158,7 +158,7 @@ impl From<HoghookCompletedRow> for AppMetric2 {
             app_source_id: row.app_source_id,
             instance_id: None,
             metric_kind: app_metrics2::Kind::Success,
-            metric_name: "".to_owned(),
+            metric_name: "Success".to_owned(),
```

Comment:
In the error case, I put the error type.

In the success case, as Hoghook only does one thing, I have no clue what to put here.

Reply:
I think it should be an `Option<String>` in that case - if nothing makes sense to put there in the success case, call it an `error_code`?

I know you currently put "Fetch" in there, but since fetching is the only fallible action hog functions can take (as far as I know?), it leaves the impression you're going to start reporting "your hog function successfully slept" or something, which seems weird